Documentatie/EnergielabelApi-validaties.md

```diff
@@ -103,28 +104,27 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
-|					|	Vergunningsaanvraag/Oplevering/bestaand	|	Wanneer er een certificaat gevonden is gelden de volgende regels:<br/><ul><li>De huidige datum moet in de geldigheidperiode liggen van het bestaande certificaat (tussen Opnamedatum en Geldig tot).</li></ul> <b>Aanvullende validaties (gelden niet voor Beheerders): </b><br/> <ul><li> Het huidige certificaat moet geregistreerd zijn door de, huidige ingelogde, gebruiker van de EnergielabelApi.</li><li>De registratiedatum van het bestaande certificaat mag maximaal een aantal dagen in het verleden liggen (deze periode is anders voor vergunningsaanvraag ten opzichte van oplevering of bestaand), of de ingelogde gebruiker heeft de rol ‘Certificaat Vervanger’ en de verloopdatum van deze rol bij de gebruiker is ingevuld en nog niet verstreken.</li></ul>
-|	Uitbreiden		|											|	**Situatie met opnamegebouw o.b.v. adres:**	<br/><ul><li> Op de opnamedatum (SurveyDate) dient er al een opnamegebouw te bestaan met dezelfde scope en gebouwklasse, met minimaal dezelfde adressen van het opnamegebouw.</li><li> Op de opnamedatum (SurveyDate) dient er al een certificaat te zijn met dezelfde scope en gebouwklasse, waarbij een adres overeenkomt met een adres van het opnamegebouw. </li></ul> **Situatie met opnamegebouw o.b.v. ProvisionalId:** <br/><ul><li> Op de opnamedatum (SurveyDate) dient er al een opnamegebouw te bestaan met dezelfde scope en gebouwklasse, met minimaal dezelfde adressen van het opnamegebouw.</li><br/><li> Op de opnamedatum (SurveyDate) dient er al een certificaat te zijn met dezelfde scope en gebouwklasse, waarbij een adres overeenkomt met een adres van het opnamegebouw. </li></ul> **Geldt voor alle situaties:** <br/><ul><li> De rekenmethodiek van het bestaande certificaat is gelijk aan die van het registratiebestand. </li><li> Er worden <b>geen</b> gebouwen toegevoegd die nog niet in het bestaande certificaat zitten, maar waarvoor wel al een ander certificaat bestaat. </li><li> Er zijn gebouwen in het registratiebestand die nog niet bij het bestaande certificaat aanwezig zijn. </li></ul>
+|:------------------|:------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+|	Toevoegen	|	<ul><li>Vergunningsaanvraag</li><li>Oplevering</li><li>Bestaand</li></ul>	|<ul><li> Toevoegen is alleen mogelijk wanneer er op de opnamedatum voor geen enkele van de opnameadressen al een registratie is met dezelfde scope en gebouwklasse.</li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ is alleen mogelijk wanneer één van de blokken ‘BAGIdentification’ of ‘ProvisionalIdentification’ gekozen is. </li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ is niet mogelijk met een ‘provisionalid’. </li><li> Toevoegen voor woningbouw met een detailaanduiding is alleen mogelijk met rechten. Deze kunt u aanvragen in EP-Online. </li></ul>
```

Comment:
Toevoegen voor de status ‘Vergunningsaanvraag’ is niet mogelijk wanneer het veld ‘provisionalid’ gevuld is.